src/cmd/config.rs

```diff
@@ -0,0 +1,39 @@
+
+use crate::{
+    cli::{ConfigCommand, ConfigAction},
+    cmd::Command,
+    conventional::Config,
+    error::Error,
+};
+
+
+impl ConfigCommand {
+    fn as_yaml_string(&self, config: &Config) -> Result<String, Error> {
+        Ok(serde_yaml::to_string(config)?)
+    }
+}
+
+impl Command for ConfigCommand {
+    fn exec(&self, config: Config) -> anyhow::Result<()> {
+        println!("{}",self.as_yaml_string(&config)?);
+        Ok(())
+    }
+}
```

Comment:
```suggestion
impl ConfigCommand {
    fn write_yaml(&self, config: &Config, w: impl Write) -> Result<(), Error> {
        Ok(serde_yaml::to_writer(w, config)?)
    }
}

impl Command for ConfigCommand {
    fn exec(&self, config: Config) -> anyhow::Result<()> {
        self.write_yaml(&config, stdout().lock())?;
        Ok(())
    }
}
```

if we ever decide to write down the config to a file this will be easier